mip/driver_tm4c.c

```diff
@@ -1,5 +1,6 @@
 #include "mip.h"
 
+#ifndef MG_ENABLE_DRIVER_IMXRT1020
```

Comment:
and we don't need this, this driver is disabled by default

Reply:
Name clashes with enum/phy accessors. Driver enum/functions need be unique'd.
Is that wishable ?

Reply:
This driver is only enabled when selected, and won't be selected with the imx driver at the same time; I don't see why

Reply:
It could be left enabled as its IRQ handler has a different name, we'll cross that bridge when we get there,

Reply:
Driver's included as soon as MG_ENABLE_MIP. 2 other drivers have name collisions This happens with enums and phy_read phy_write functions (stm32+w5500 come with MIP definition).

Drivers can go with prefix or unique inclusion way. I opted for the second, but i also drafted the 1st solution.
This was discussed a bit. I'll leave it open for the moment. This commit isn't pushed.